lib/Controller.js

```diff
@@ -253,14 +259,18 @@ module.exports = (function(){
             return Q.all(stopPromises);
         }
     };
-
+    
     Controller.prototype.getStat = function() {
         return {
             "handledEntries": this.handledEntries,
             "phantomEntries": this.phantomEntries
         }
     };
 
+    Controller.prototype.getAdapter = function() {
```

Comment:
removed getAdapter from Controller.js and updated AdapterFactory to support only one timer adapter instantitiation.